test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -384,7 +384,7 @@ describe('CSOT spec prose tests', function () {
         clock.restore();
       });
 
-      it('serverSelectionTimeoutMS honored if timeoutMS is not set', async function () {
+      it.skip('serverSelectionTimeoutMS honored if timeoutMS is not set', async function () {
```

Comment:
Instead of skipping these tests, can't we just call `connect()` manually on the client before performing the operation that is using timeoutMS?

Reply:
No because `connect` throws unfortunately, and if you squash the error then you get a client unconnected error